pkg/util/openstack/loadbalancer.go

```diff
@@ -42,6 +42,7 @@ const (
 	OctaviaFeatureFlavors           = 2
 	OctaviaFeatureTimeout           = 3
 	OctaviaFeatureAvailabilityZones = 4
+	OctaviaFeatureHTTPMonitorsOnUDP = 8
```

Comment:
Unless I'm missing something, we're doing arithmetic based on these constants so it should be safe to have picked `5`.

Reply:
I have no idea why I missed there's a `3` in there already, thanks for catching this. :smiley: